docs/source/en/model_doc/mms.mdx

```diff
@@ -30,21 +30,24 @@ for the same number of languages, as well as a language identification model for
 Experiments show that our multilingual speech recognition model more than halves the word error rate of 
 Whisper on 54 languages of the FLEURS benchmark while being trained on a small fraction of the labeled data.*
 
+Here are the different models open sourced in the MMS project. The models and code are originally released [here](https://github.com/facebookresearch/fairseq/tree/main/examples/mms). We have add them to the `transformers` framework, making them easier to use.
```

Comment:
We mention the original repository below, so not strictly necessary to re-reference it here!

Reply:
I removed the other reference.